Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -1986,6 +1986,7 @@ RefPtr<Gfx::FontCascadeList const> StyleComputer::compute_font_for_style_values(
         case Keyword::Monospace:
         case Keyword::UiMonospace:
             generic_font = Platform::GenericFont::Monospace;
+            font_size_in_pt = 13 * 0.75f;
```

Comment:
So if we hit `find_generic_font` you override the calculated font size? That doesn't seem like the right solution; shouldn't you change the default font size of `16` at line 1814?

Reply:
if I change that from `16` to `13` it'll change it for all fonts but we only want to change it for monospaced fonts (`i.e font-family: monospace`)

Reply:
could do something like this so we don't override it until the very end

```c++
auto found_font = StyleProperties::font_fallback(monospace, bold);
font_size_in_pt = monospace ? 13 * 0.75f : font_size_in_pt;
font_list->set_last_resort_font(found_font->with_size(font_size_in_pt));
```

Reply:
or yeah would need a way to modify `CSSPixels font_size_in_px = 16;` for only monospaced font's but we don't know that at that point

Reply:
It might help if you can find if this is something that is in a spec somewhere, or if it's custom behavior by both Chrome and Firefox (user agent stylesheet?), etc. Just changing something because WPT expects it is not always the right course of action; e.g. maybe their assumption is wrong and we need to report an issue?

Reply:
it's a fair point, I did search the default css in the spec and there is no mention of it, nothing online either. I can only see that this is the behaviour on Firefox and Chrome (if you go into appearance settings it's set there). That's what the FIXME on the line you speak of is working towards I believe (letting the user specify).

Can you please point me to where I'd open an issue or investigate this? In the meantime I can have a search around the WPT tests to see where they specify `font-family: monospace` and see if any rely on calculations based off of a default value of `13px` for this font, which like you've said is probably an assumption to accomodate current existing behaviour that actually isn't in any spec.

Reply:
Right, the FIXME is to support user font sizes, and WPT should probably never rely on those.

Reply:
I've opened an issue https://github.com/web-platform-tests/wpt/issues/48840 will close this for now and see what comes out of that.